userspace/libsinsp/filterchecks.h

```diff
@@ -300,7 +301,7 @@ class sinsp_filter_check_fd : public sinsp_filter_check
 
 private:
 	uint8_t* extract_from_null_fd(sinsp_evt *evt, OUT uint32_t* len, bool sanitize_strings);
-	bool extract_fdname_from_creator(sinsp_evt *evt, OUT uint32_t* len, bool sanitize_strings);
+	bool extract_fdname_from_creator(sinsp_evt *evt, OUT uint32_t* len, bool sanitize_strings, bool fd_nameraw);
```

Comment:
I think that defaulting to false here would save quite a bit of changes, thus reducing the diff :)